### DISCUSSION TITLE
How To Change NavigationView's TogglePaneButton Default Icon ?

### DISCUSSION THREAD

**Discussion by OmerFarukAkyapak** [amwx/FluentAvalonia#533] [Q&A] [answered]
I can change to navigation view item's icon but i cant change to  navigation view's default icon. i want to change it because my icon doesnt look like on fluent sample project. Why my icon dosnt look like fluent sample's icon, or how to change this icon ?

Windows Version : 10
Avaloina Version : 11.0.9
Fluent Avalonia Version : 2.0.5

My Project :

![image](https://github.com/amwx/FluentAvalonia/assets/56655714/0ed70a24-a1ef-4910-be28-564bf90aa525)

Fluent Sample Project :

![image](https://github.com/amwx/FluentAvalonia/assets/56655714/025aaad8-30d4-407c-82b2-c79720688132)

Regards...

**Comment by huseyinhamad** [comment]
Same issue on macOS. Got question mark as icon instead of sample projects' icon

**Comment by OmerFarukAkyapak** [comment, marked as answer]
There are conflicts over Avalonia theme packs.
When I edited the code line as below and removed the style I added, it was fixed.

![image](https://github.com/amwx/FluentAvalonia/assets/56655714/983e4a82-61bf-4f4d-9e49-74efc5369d3e)

**Reply by robloo** [reply]
You probably shouldn't be using `<FluentTheme />` AND `<FluentAvaloniaTheme />` at the same time

**Reply by OmerFarukAkyapak** [reply]
Yes it does but how do I use the other controls when I remove FluentTheme ? (like datagrid)
 
```
 <Application.Styles>
	
	<FluentTheme />
	<sty:FluentAvaloniaTheme />
	<StyleInclude Source="avares://AvaloniaEdit/Themes/Fluent/AvaloniaEdit.xaml"/>
	<StyleInclude Source="avares://FluentAvalonia/Styling/ControlThemes/Controls.axaml"/>
	<StyleInclude Source="avares://Avalonia.Controls.DataGrid/Themes/Fluent.xaml"/>
	
</Application.Styles>
```

**Reply by timunie** [reply]
DataGrid should be there, AvaloniaEdit: You should adopt the style to match FA
